paper/examples/flagstat/flagstat.go

```diff
@@ -0,0 +1,110 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
```

Comment:
This line must be blank (no comment at all); godoc will render this as two paragraphs.